Add smoketest (package installed, keyring/repo files in place)

rocodes opened the pull request:
Closes #8 

- Add test that keyring/repo files are in place and rpmdb contains key (todo: integrate into CI so that package is built and installed into a Fedora container and then smoketests run)
- Update README to explain rpm key id and link to rpm release management docs

legoktm commented on tests/test_keyring.py:
```diff
@@ -0,0 +1,52 @@
+# Basic acceptance testing for keyring package.
+# Expects fedora environment.
+
+RPM_QUERY_KEYRING_PACKAGE = ["rpm", "-q", "securedrop-workstation-keyring"]
```
Unless these need to be reused, the tests would be easier to read if the commands were inlined where they get invoked.

rocodes replied:
(reuse it a bit checking multiple different packages, but made it clearer what's happening at invocation)

legoktm commented on tests/test_keyring.py:
```diff
@@ -0,0 +1,52 @@
+        # exit code 1 if package is not installed
+        subprocess.check_call(RPM_QUERY_KEYRING_PACKAGE)
+        return True
+    except subprocess.CalledProcessError:
```
Consider letting the exception bubble up; it will still cause the test to fail and probably provides a little more output about what failed instead of "asserting False failed".

legoktm reviewed (commented):
Two small comments about the tests, but overall looks good!

rocodes commented:
Thank you for the feedback, if only I had pushed my extra changes bit min sooner! I've added support for running the tests in the container, addressed your comment re test failure, and (wip) adding github action support.

rocodes commented:
(The way I'm trying to shove systemd into the fedora container is unsurprisingly not ideal:  `System has not been booted with systemd as init system (PID 1). Can't operate.` :upside_down_face:  I'll look into this tomorrow)

eloquence commented:
@rocodes Is this still needed with the latest version of the package?
